aesd-char-driver/main.c

```diff
@@ -121,9 +253,14 @@ void aesd_cleanup_module(void)
 
     cdev_del(&aesd_device.cdev);
 
-    /**
```

Comment:
What happens to `partial_write_buff` if a partial write was in progress before cleanup?